guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,57 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {EL} 7 to {EL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
+
+If you have custom Ansible content such as playbooks, job templates inside REX, roles and collections on disk, and you relied on modules being delivered by the Ansible RPM on {Project}, you have to take additional steps to adapt your Ansible installation or migrate your Ansible content.
+
+Ansible Core removed all non-essential modules shipped in the core installation.
+In terms of FQCN notation `namespace.collection.module`, you can continue to use `ansible.builtin.*`, but everything else is missing in Ansible Core.
+That means you will be no longer able to use non-builtin Ansible modules as you were used to and you have to get them from somewhere else, eventually.
+
+Current options to resolve this situation:
+
+* You can download and install additional community-maintained modules from Galaxy server.
```

Comment:
```suggestion
* You can obtain additional community-maintained collections which provide the removed functionality from Ansible Galaxy.
```

* "download and install" → "obtain" is a personal choice of mine (the users don't need to do the download manually, the `ansible-galaxy` tool will do it for them), so change as you see fit
* "modules" → "collections" as that's the more appropriate term for the entity that is installed
* "which provide the removed functionality" - I think that's an important detail ;)